Add Enhance Content Section to Help Center (#124)
bump-sh/docs: src/_help/enhance-documentation-content/topics.md

---
title: Topics
---

- TOC
{:toc}

> Topics are an exclusive custom property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.

Reviewer: Is this working both for OpenAPI and AsyncAPI?

Reviewer 2: Yes it does 👍

> Topics are an exclusive custom property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.

Reviewer 3: ```suggestion
`x-topics` is a vendor-specific property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.
```

Author: For my own curiosity: why "vendor-specific"? What is a vendor?

> By default, most specification standards do not support the addition of extra content. So we have added an additional property to insert at the beginning of your specification file: `x-topics`.

Reviewer 3: I think we can remove this sentence following my previous proposal, as we directly mention the property. I find it more clear like this, WDYT?

> By default, most specification standards do not support the addition of extra content. That's why we've added this additional property to insert at the beginning of your specification file: `x-topics`.

Reviewer 2: ```suggestion
By default, most specification standards do not support the addition of extra content, so we added this additional property to insert at the beginning of your specification file: `x-topics`.
```

Author: Not 100% sure of the transition here and that would make a super long sentence.

Reviewer 2: That was just a suggestion because the rhythm of that part sounded a bit weird to me but you make the call. :)

## Use `x-topics`

|Property|Description|
|---|---|
|title|Topic title as it will appear in the navigation bar and in the content section.|
|content|The topic content. Markdown is fully supported here.|
|example|Will appear in the examples section, if activated. Markdown is fully supported here.|

Example:

```yaml
x-topics:
  - title: Getting started
    content: Before using the API you need to get an API key by sending us an email.
  - title: Authentication
    content: Send the `X-API-KEY` header with all your requests.
    example: |
      ```
      $ curl \
        -X POST https://api.example.com/endpoint/ \
        -H "X-API-KEY: XXXXXX" \
      ```
```

## Public documentation examples

Here are some examples of public documentation made by teams using `x-topics`:

- [Memo Bank API Documentation](https://docs.api.memo.bank/)

> - [Pexip Engage API Documentation](https://docs.api.memo.bank/)

Reviewer 3: Bad URL
